lib/pyfrc/mains/cli_create_physics.py

```diff
@@ -0,0 +1,126 @@
+        bumper_width = 3.25 * units.inch
+
+        self.drivetrain = tankmodel.TankModel.theory(
+            motor_cfgs.MOTOR_CFG_CIM,           # motor configuration
+            110 * units.lbs,                    # robot mass
+            10.71,                              # drivetrain gear ratio ωIn/ωOut
+            2,                                  # motors per side„
+            22 * units.inch,                    # robot wheelbase
+            23 * units.inch + bumper_width * 2, # robot width
+            32 * units.inch + bumper_width * 2, # robot length
+            6 * units.inch,                     # wheel diameter
+        )
+        """
+
+    def update_sim(self, hal_data, now, tm_diff):
+        """
+            Called when the simulation parameters for the program need to be
+            updated.
+            
+            :param now: The current time as a float
+            :param tm_diff: The amount of time that has passed since the last
+                            time that this function was called
+        """
+
+        # Simulate the drivetrain
+        """
+        lr_motor = hal_data["pwm"][1]["value"]
+        rr_motor = hal_data["pwm"][2]["value"]
+
+        # Not needed because front and rear should be in sync
+        # lf_motor = hal_data['pwm'][3]['value']
+        # rf_motor = hal_data['pwm'][4]['value']
+
+        x, y, angle = self.drivetrain.get_distance(lr_motor, rr_motor, tm_diff)
+        self.physics_controller.distance_drive(x, y, angle)
+        """
+'''
+
+sim_starter = {
+    "pyfrc": {
+        "robot": {
+            "w": 3,
+            "h": 2,
+            "starting_x": 10,
+            "starting_y": 10,
+            "starting_angle": 0,
+        },
+        "pwm": {},
+        "can": {},
+        "joysticks": {
+            "0": {"axes": {}, "buttons": {}},
+            "1": {"axes": {}, "buttons": {}},
+        },
+    }
+}
+
+
+class PyFrcCreatePhysics:
+    def __init__(self, parser=None):
+        pass
+
+    def run(self, options, robot_class, **static_options):
+        robot_file = abspath(inspect.getfile(robot_class))
+        robot_path = dirname(robot_file)
+        sim_path = join(robot_path, "sim")
+
+        physics_file = join(robot_path, "physics.py")
+        sim_file = join(sim_path, "config.json")
+        if exists(physics_file):
+            print("- physics.py already exists")
+        else:
+            with open(physics_file, "w") as fp:
+                fp.write(physics_starter)
+            print("- physics file created at", physics_file)
+
+        if exists(sim_path):
+
+            if exists(sim_file):
+                print("- config.json already exists")
+            else:
+                with open(sim_file, "w") as f:
+                    f.write(sim_starter)
+                print("- sim file created at", sim_file)
+        else:
+            mkdir(sim_path)
+            with open(sim_file, "w") as f:
+                f.write(json.dump(sim_starter))
```

Comment:
This... isn't how you use json.dump. Also, you will want to pretty-print it as well.